spec/acceptance/self_service_spec.rb

```diff
@@ -15,7 +15,7 @@
     # Test Confirms all facts are false which is another indicator the class is performing correctly
     describe 'check no self_service fact is false' do
       it 'if idempotent all facts should be true' do
-        expect(host_inventory['facter']['self_service'].size).to eq(14)
+        expect(host_inventory['facter']['self_service'].size).to eq(16)
```

Comment:
Just be careful with this one! its correct now, but depending on when or who adds in a PR before this one is merged you may have to increment again